stable/rabbitmq/README.md

```diff
@@ -110,7 +110,7 @@ $ helm install --name my-release -f values-production.yaml stable/rabbitmq
 
 ## Persistence
 
-The [Bitnami RabbitMQ](https://github.com/bitnami/bitnami-docker-rabbitmq) image stores the RabbitMQ data and configurations at the `/bitnami/rabbitmq` path of the container.
+The [Bitnami RabbitMQ](https://github.com/bitnami/bitnami-docker-rabbitmq) image stores the RabbitMQ data and configurations at the `/opt/bitnami/rabbitmq/var/lib/rabbitmq/` path of the container.
```

Comment:
now we only persist data but not configurations, right?

Reply:
correct, it persist automatic configurations created by the tool and we overright erlang cookie and config on the entry point script

Reply:
so technically still persisted, So I wouldnt change the comment

Reply:
ok!